docs/materials/day4/part2-ex2-file-transfer.md

```diff
@@ -49,49 +47,51 @@ queue
 
 ### Implement file compression
 
-In our first blast job from yesterday, the database files in the `pdbaa` directory were all transferred, as is, but we could instead transfer them as a single, compressed file using `tar`. For a second test job, let's compress our blast database files to send them to the submit node as a single `tar.gz` file, by following the below steps
+In our first blast job from yesterday, the database files in the `pdbaa` directory were all transferred, as is, but we could instead transfer them as a single, compressed file using `tar`. For a second test job, let's compress our blast database files to send them to the submit node as a single `tar.gz` file, by following the below steps:
 
 1. Change into the `pdbaa` directory and compress the database files into a single file called `pdbaa_files.tar.gz` using the `tar` command. (NOTE: This file will be different from the `pdbaa.tar.gz` files you downloaded yesterday, because it will only contain the `pdbaa` files, and not the `pdbaa` directory, itself.)
 
-A typical command for creating a tar file is:
-
-``` console
-user@user-training $ <strong>tar -cvzf [compressed filename] [list of files]</strong>
-```
+    A typical command for creating a tar file is:
 
-Move this file to the `thur-blast-data` directory.
+        :::console
+        user@training $ tar cvzf [compressed filename] [list of files]
```

Comment:
Angle brackets instead of square brackets, all caps. "LIST OF FILES OR DIRECTORIES"